app/views/update_checks/graphs.html.erb

```diff
@@ -14,7 +14,10 @@
 
 <h4>Absolute numbers 🚀</h4>
 <ul>
-<li>Total number of <i>fastlane</i> launches: <b><%= number_with_delimiter(@cumulative.find { |a| a[:label] == "fastlane" }[:data].last) %></b></li>
+<% fl_data = @cumulative.find { |a| a[:label] == "fastlane" } %>
+<% if fl_data %>
+  <li>Total number of <i>fastlane</i> launches: <b><%= number_with_delimiter(@cumulative.find { |a| a[:label] == "fastlane" }[:data].last) %></b></li>
```

Comment:
`fl_data = @cumulative.find { |a| a[:label] == "fastlane" }` is used two lines above, so:

`number_with_delimiter(@cumulative.find { |a| a[:label] == "fastlane" }[:data].last)` -> `number_with_delimiter(fl_data[:data].last)`